2996_reflection/reflection.md

```diff
@@ -82,6 +82,14 @@ Other advantages of a single opaque type include:
     nontypes — without fear of slicing values).
 
 
+## Implementation Status
+
+Lock3 implemented the equivalent of much that is proposed here in a fork of Clang (specifically, it worked with the P1240 proposal, but also included several other capabilities including a first-class injection mechanism).
```

Comment:
Can you add a compiler explorer link here? I never remember where it actually is.